dev/MRTCore/mrt/Microsoft.ApplicationModel.Resources/src/Microsoft.ApplicationModel.Resources.idl

```diff
@@ -54,6 +54,10 @@ namespace Microsoft.ApplicationModel.Resources
         IKeyValuePair<String, ResourceCandidate> GetValueByIndex(UInt32 index);
         [method_name("GetValueByIndexWithContext")]
         IKeyValuePair<String, ResourceCandidate> GetValueByIndex(UInt32 index, ResourceContext context);
+
+        ResourceCandidate TryGetValue(String resource);
```

Comment:
Should these go into an `MrtContract, 1.1` since it's changing the public surface area in a nonbreaking fashion?

Reply:
I thought it was ok to keep the contract due to not having official release. If it's not ok, I can add a new contact. What do you think?

Reply:
@EHO-makai - what do _you_ think? It's probably OK to leave this at the 1.0 version for now, given our current conversations about release vehicles and versions.

Reply:
That's probably what I would do.  We should probably take advantage of the relaxed rules around breaking changes for 0.* release.  It would be a bit odd to arrive at Reunion 1.0 and have MrtContract 1.6.  :-)

Reply:
> take advantage of the relaxed rules around breaking changes for 0.* release

Link to doc?